packages/node_modules/pouchdb-adapter-idb/src/index.js

```diff
@@ -379,6 +379,11 @@ function init(api, opts, callback) {
     var type = attachment.content_type;
 
     txn.objectStore(ATTACH_STORE).get(digest).onsuccess = function (e) {
+      if (!e.target.result && attachment.unfetched) {
```

Comment:
Same as above

Reply:
I originally had it there. But this for the case we replicate back to the original source db. Then we can remove the unfetched flag since the attachment exists.